### DISCUSSION TITLE
Slideshow speed

### DISCUSSION THREAD

**Discussion by wyliecoyoteuk** [jellyfin/jellyfin#9528] [Q&A] [answered]
HI, is there any way to slow down the slideshow, it runs so quickly that some images don't finish loading.
Is there a variable I can change?

**Comment by Jarige** [comment]
Also interested in this!

**Comment by wyliecoyoteuk** [comment]
Sadly, nobody else seems to be.

**Comment by vipsnake** [comment]
I realise I'm a little late to the party, however I just started using Jellyfin, seems really great, however one feature that would be great is to set the duration a photo shows during a slideshow, before moving to the next. This is the main reason I started with Jellyfin (viewing photos), so would be great if this can be addressed.

As these are also memories, it would be great to stick on an image for some time, before it continues (eg. 5mins or even 30 mins)

Thanks

**Comment by Skalmansbror** [comment]
Please, developers, just add screensaver options for the user to adjust the delay...
This shouldn't be too hard to implement.
I've donated more to jellyfin than a plex life time pass costs, so I feel entitled as heck ;)

**Comment by ZombieDriver** [comment]
This would be incredibly useful. 

At the moment it's just too fast to appreciate photos - as soon as one loads and you start to look at it, it's moving on to the next one....

**Reply by wyliecoyoteuk** [reply]
I agree. In fact It often fails to load an image fully before skipping to the next.
as it stands, the slideshow function is useless.

**Comment by Jarige** [comment]
I'm glad there's some activity here and other people that are interested. I've searched for the code that's responsible for the slideshow but the amount of changes was too much for me to create the feature. Mostly because I am not familiar with the codebase and it's a tad bit more complicated than a few lines of code in one file. But for someone who's familiar with the code, I'd estimate this not to be complicated. The library that's used for the slideshow is quite configurable so it should be possible. I created my own build of Jellyfin once where I changed the speed, but it was hardcoded and the effort to make it configurable or to make GUI items was too much for the amount of time I had.

Let's hope someone is willing roads this for us!

**Reply by demuxer86** [reply]
any updates?  this is a feature that basically none of the slideshow apps have, at least ones that i have tried.  its maddening.    i actually think i can just make a tasker command that hits the pause button every 30 seconds :p

**Comment by wyliecoyoteuk** [comment]
On my Jellyfin server which sits in a corner of the Kitchen, I run Xscreensaver for a slideshow.
I run a Jellyfin client on an old Sony Android tablet in the living room, but I have to run a separate app for a slideshow.
I think the slideshow functionality in Jellyfin is obviously an afterthought.
In fact it is so poor, it might as well be removed.

**Comment by nielsvanvelzen** [comment, marked as answer]
We don't use GitHub discussions to track feature requests for the server/web interface. Please find or open a request at features.jellyfin.org.
